packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -358,7 +358,7 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
         ) {
           const { priceBook, curAuctionPrice } = this.state;
           const { helper } = this.facets;
-          trace('acceptPrice');
+          trace('acceptPrice', this.state.collateralBrand);
```

Comment:
consider putting it at the beginning for hierarchy in the logs (Vault manager: brand: unique message)

Reply:
a fine idea.